src/initialization.py

```diff
@@ -3,13 +3,21 @@
 import re
 import sys
 
+from jnius import autoclass
+from android_utils import get_activity
 
 script_dir = os.path.dirname(os.path.abspath(__file__))
 sys.path.append(script_dir)
 sys.path.append(os.path.join(script_dir, "kolibri", "dist"))
 sys.path.append(os.path.join(script_dir, "extra-packages"))
 
 os.environ["DJANGO_SETTINGS_MODULE"] = "kolibri_app_settings"
+Secure = autoclass('android.provider.Settings$Secure')
+
+os.environ["MORANGO_NODE_ID"] = Secure.getString(
+    get_activity().getContentResolver(),
+    Secure.ANDROID_ID
+)
```

Comment:
It looks like there's at least one special ID we want to handle (not use): https://groups.google.com/g/android-developers/c/U4mOUI-rRPY

So if the value is "9774d56d682e549c", and perhaps also if it's shorter than 16 characters, we should skip setting the envvar.